src/Sylius/Bundle/CartBundle/EventListener/RefreshCartListener.php

```diff
@@ -11,24 +11,45 @@
 
 namespace Sylius\Bundle\CartBundle\EventListener;
```

Comment:
This should be moved to Doctrine folder as it's no longer Symfony event listener but Doctrine one.